### ISSUE TITLE
goreleaser v2.4.6 does not show any error logs

### ISSUE THREAD

**Issue by wojiushixiaobai** [goreleaser/goreleaser#5292]
### What happened?

```sh
docker run --rm -it golang:1.23-bookworm bash
go install github.com/goreleaser/goreleaser/v2@latest
goreleaser -v
  ____       ____      _
 / ___| ___ |  _ \ ___| | ___  __ _ ___  ___ _ __
| |  _ / _ \| |_) / _ \ |/ _ \/ _` / __|/ _ \ '__|
| |_| | (_) |  _ <  __/ |  __/ (_| \__ \  __/ |
 \____|\___/|_| \_\___|_|\___|\__,_|___/\___|_|
goreleaser: Deliver Go Binaries as fast and easily as possible
https://goreleaser.com

GitVersion:    v2.4.6
GitCommit:     unknown
GitTreeState:  unknown
BuildDate:     unknown
BuiltBy:       unknown
GoVersion:     go1.23.3
Compiler:      gc
ModuleSum:     h1:RAuIc/BIKifjHq42+b97RdU1BQMoJKA9zuKiOh8Yjls=
Platform:      linux/arm64
```
```sh
goreleaser release
# does not show any error logs
```

### How can we reproduce this?

```sh
docker run --rm -it golang:1.23-bookworm bash
go install github.com/goreleaser/goreleaser/v2@latest

# or
wget https://github.com/goreleaser/goreleaser/releases/download/v2.4.6/goreleaser_Linux_arm64.tar.gz
tar -xf goreleaser_Linux_arm64.tar.gz -C /usr/local/bin goreleaser

```sh
goreleaser release
```

### goreleaser version

```bash
goreleaser -v
  ____       ____      _
 / ___| ___ |  _ \ ___| | ___  __ _ ___  ___ _ __
| |  _ / _ \| |_) / _ \ |/ _ \/ _` / __|/ _ \ '__|
| |_| | (_) |  _ <  __/ |  __/ (_| \__ \  __/ |
 \____|\___/|_| \_\___|_|\___|\__,_|___/\___|_|
goreleaser: Deliver Go Binaries as fast and easily as possible
https://goreleaser.com

GitVersion:    v2.4.6
GitCommit:     unknown
GitTreeState:  unknown
BuildDate:     unknown
BuiltBy:       unknown
GoVersion:     go1.23.3
Compiler:      gc
ModuleSum:     h1:RAuIc/BIKifjHq42+b97RdU1BQMoJKA9zuKiOh8Yjls=
Platform:      linux/arm64
```

### GoReleaser Check

- [X] goreleaser check shows no errors

### Search

- [X] I did search for other open and closed issues before opening this

### Supporter

- [ ] I am a [sponsor](https://github.com/sponsors/caarlos0/) or a [Pro](https://goreleaser.com/pro) customer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wojiushixiaobai** [comment]
![image](https://github.com/user-attachments/assets/c77e06c2-1e90-43ed-86c9-6963b7217692)

**Comment by caarlos0** [comment]
that's weird 🤔 

started happening after #5263 - which changed mostly test code...
 https://github.com/goreleaser/goreleaser/actions/runs/11870499251/job/33081972871#step:18:343

trying to figure it out!


thanks for the report, i hadn't noticed the lack of output 🤔

**Comment by caarlos0** [comment]
fixed in 2.4.7!

thanks for reporting this! ❤️
